impl/client/src/main/javascript/web/repo/pentaho-ajax.js

```diff
@@ -14,334 +14,309 @@
 * limitations under the License.
 *
 */
-define("common-repo/pentaho-ajax",[], function(){});
+define("common-repo/pentaho-ajax",["common-repo/pentaho-csrf"], function(){});
+
+// Project: pentaho-platform-plugin-common-ui
+
+/* globals CONTEXT_PATH, FULL_QUALIFIED_URL, WEB_CONTEXT_BASE */
+
```

Comment:
Almost exactly what I meant in the dashboards review :-)
Don't need to create a sync csrf/service, though.